style.css

```diff
@@ -0,0 +1,1043 @@
+:root {
+  --color-black: rgb(0, 0, 0);
+  --color-grey: rgb(219, 216, 216);
+  --color-light-grey: rgb(230, 230, 230);
+  --color-white: rgb(255, 255, 255);
+  --chakra-font: 'Chakra Petch', sans-serif;
+  --color-ligth-green: rgb(0, 204, 204);
+  --color-orange: rgb(255, 68, 34);
+  --color-yellow: rgb(255, 221, 51);
+}
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100%;
+  width: 100%;
+  overflow-x: hidden;
+  background: var(--color-white) center center fixed;
+  background-size: cover;
+}
+
+li {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+footer ul {
+  display: flex;
+  justify-content: space-evenly;
+  width: 25rem;
+}
+
+footer i {
+  width: 100%;
+  height: 100%;
+}
+
+footer a {
+  color: var(--color-white);
+  height: 1rem;
+}
+
+.footer-media li {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 3rem;
+  height: 3rem;
+  border-radius: 50%;
+  background-color: rgba(75, 73, 73, 0.5);
+}
+
+.footer-section-2 div {
+  display: flex;
+}
+
+.uppercase {
+  text-transform: uppercase;
+}
+
+.nav-1 {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  position: relative;
+  background-color: var(--color-black);
+  width: 100%;
+  height: 2rem;
+  color: var(--color-white);
+}
+
+header {
+  position: sticky;
+  top: 0;
+  z-index: 10;
+}
+
+header p {
+  background-color: var(--color-yellow);
+  width: 100%;
+  padding: 0.5%;
+  font-size: 0.9rem;
+  margin-top: 0;
+}
+
+header p a {
+  color: var(--color-black);
+}
+
+.nav-2-ul-1 a {
+  display: block;
+  color: var(--color-black);
+  opacity: 0.5;
+  min-width: 5%;
+  text-align: center;
+  margin-top: 1rem;
+  text-transform: uppercase;
+  font-family: var(--chakra-font);
+  font-size: 0.8rem;
+}
+
+.mid-article-head-title a {
+  color: var(--color-orange);
+}
+
+.nav-1 li {
+  margin-right: 3%;
+  opacity: 0.5;
+}
+
+.mid-article-head-title li {
+  margin-right: 1rem;
+}
+
+.main-articles-footer a {
+  color: var(--color-grey);
+  font-size: 0.7rem;
+}
+
+.nav-1 a {
+  color: var(--color-white);
+}
+
+.latest-news-grid a {
+  color: var(--color-black);
+}
+
+.mid-article-head-content-1-body a {
+  color: grey;
+  font-size: 0.9rem;
+}
+
+.article-content-2 ul li {
+  list-style: disc;
+  color: var(--color-orange);
+  margin-left: 0.2rem;
+  border-bottom: 1px solid var(--color-light-grey);
+  padding: 0.6rem 0;
+}
+
+.latest-deals-grid-header a {
+  color: var(--color-orange);
+}
```

Comment:
This selector and selector on line 110 have the same style definition, group selectors to minimize code.

Reply:
Implemented this